sdk/eventgrid/eventgrid/README.md

```diff
@@ -1,119 +1,210 @@
-## Azure EventGridClient SDK for JavaScript
+# Azure Event Grid client library for JavaScript
 
-This package contains an isomorphic SDK for EventGridClient.
+[Azure Event Grid](https://azure.microsoft.com/en-us/services/event-grid/) is a cloud-based service that provides reliable event delivery at massive scale.
+
+Use the client library to:
+
+- Send events to Event Grid using either the Event Grid, Cloud Events 1.0 schemas, or a custom schema
+- Decode and process events where were delivered to an Event Grid handler
+- Generate Shared Access Signatures for Event Grid topics
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventgrid/eventgrid/) |
+[Package (NPM)](https://www.npmjs.com/package/@azure/eventgrid) |
+[API reference documentation](https://aka.ms/azsdk-js-eventgrid-ref-docs) |
+[Product documentation](https://docs.microsoft.com/en-us/azure/event-grid/) |
+[Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventgrid/eventgrid/samples)
+
+## Getting started
 
 ### Currently supported environments
 
-- Node.js version 6.x.x or higher
-- Browser JavaScript
```

Comment:
should we call out that we dropped browser support?

Reply:
It was intentional to say we had dropped browser support (this is likely due to be starting from an existing README.md from another SDK and maybe it didn't support the browser).  I will add back that we support it.